docx/section.py

```diff
@@ -80,6 +83,16 @@ def gutter(self):
     def gutter(self, value):
         self._sectPr.gutter = value
 
+    @lazyproperty
+    def header(self):
+        """
+        Return the |Header| object representing the default header for this
+        section. A |Header| object is always returned, whether such a header is
+        present or not. The header itself is added, updated, or removed using the
+        returned object.
+        """
```

Comment:
All formatting guidance is specific to this project btw. I don't subscribe to religious views about that sort of thing, but I do value the speed and sharper vision that consistency within the project brings :)

* No blank line between docstring and first method.

* Docstrings in the project (and generally everything in a .py file) are wrapped to 77 (vim: :set textwidth=77). This allows a commented-out section to pass the flake8 validation. (I have that built into my MacVim btw, pretty sure it's syntastic that runs flake8 every time I write the file.)

* flake8 should also be run as part of the tests. I have 't' set up as an alias that maps to this:

  ```bash
  :( ± alias t
  t='make clean && flake8 && py.test && behave --stop'
  ```

* '@wip' comes off the acceptance test on the first commit where it passes. This time it's only the one commit. Often it's a few or handful.